projects/laji/src/app/shared/pipe/pluck.pipe.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'pluck'
+})
+export class PluckPipe implements PipeTransform {
+
+  transform<T extends any, K extends keyof T>(value: T|T[], key: K): T[K] | Array<T[K]> {
+    if (Array.isArray(value)) {
+      return value.map(v => v?.[key]);
```

Comment:
Unnecessary `?.` operator (sorry for nagging, but it's good practice to try to keep compiled code short :)

Reply:
No if the value is `['foo', null]` with out ?. that would cause an error

Reply:
Indeed... Sorry, I misread here that `v` would be the array :)